### PR TITLE
Switch to official cppnetlib

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by rdumusc** [comment]
Yeah! Good news!

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "cppnetlib"]
 	path = cppnetlib
-	url = https://github.com/BlueBrain/cpp-netlib
+	url = https://github.com/cpp-netlib/cpp-netlib
 	branch = 0.13-release
```
Comment (tribal-tec) [inline .gitmodules]: master branch?

**Comment by tribal-tec** [comment]
missing rebase

File: .gitmodules (hunk 1 above)
Comment (eile) [inline .gitmodules]: Sure?

Comment (rdumusc) [inline .gitmodules]: I'm not sure, branch 0.13-release has your latest fixes for ICC that master does not have:
https://github.com/cpp-netlib/cpp-netlib/commits/0.13-release
https://github.com/cpp-netlib/cpp-netlib/commits/master
actually why not use an exact sha1? Using a development branch might break the build at any time.

**Pull request merged**